travis-install-wx.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+rm -Rf wxPython-src-2.8.12.1
+wget http://sourceforge.net/projects/wxpython/files/wxPython/2.8.12.1/wxPython-src-2.8.12.1.tar.bz2/download
```

Comment:
Is it a good idea to download from `sourceforge` with reports of sourceforge bundling malware in the downloads?